nodejs/eks/cluster.ts

```diff
@@ -56,6 +56,12 @@ export interface ClusterOptions {
      */
     nodeRootVolumeSize?: pulumi.Input<number>;
 
+    /**
+     * Extra code to run on node startup. This code will run after the AWS EKS bootstrapping code and before the node
+     * signals its readiness to the managing CloudFormation stack.
+     */
+    nodeUserData?: pulumi.Input<string>;
```

Comment:
Perhaps include some version of this point in the docs - it isn't obvious that this would be a requirement:

> this code must be formatted like other user data--e.g. it must start with an interpreter line (#!)--and will be run after EKS bootrstrapping but before the node signals readiness to CF.

Reply:
Done :+1: